packages/account/src/providers/provider.ts

```diff
@@ -746,6 +758,13 @@ Supported fuel-core version: ${supportedVersion}.`
     return bn(height);
   }
 
+  /**
+   * Destroys the provider instance.
+   */
+  destroy() {
+    clearInterval(this.autoRefreshIntervalId);
+  }
```

Comment:
When destroying should we also unset the `autoRefreshIntervalId`?

```ts
this.autoRefreshIntervalId = undefined
```

Reply:
6f1bf7e69d05a48e34715551a22ed9ed58b503ce